src/styles/blocks/header.scss

```diff
@@ -0,0 +1,90 @@
+@import '../utils/variables';
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  background: $decoration-color;
+  font-weight: 500;
+  margin: 0;
+}
+
+.header {
+  background-color: $decoration-color;
+  width: 100%;
+  min-width: 1024px;
+  align-items: center;
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav {
```

Comment:
Move it to separate file as it is separate BEM block

Reply:
Done